crates/swc_html_minifier/src/lib.rs

```diff
@@ -1533,17 +1533,25 @@ impl Minifier<'_> {
             },
         );
 
+        swc_ecma_visit::VisitMutWith::visit_mut_with(
+            &mut program,
+            &mut swc_ecma_transforms_base::hygiene::hygiene(),
+        );
+        swc_ecma_visit::VisitMutWith::visit_mut_with(
+            &mut program,
+            &mut swc_ecma_transforms_base::fixer::fixer(if options.parser.comments {
+                Some(&comments)
+            } else {
+                None
+            }),
+        );
+
         let mut buf = vec![];
 
         {
-            let mut wr = Box::new(swc_ecma_codegen::text_writer::JsWriter::new(
-                cm.clone(),
-                "\n",
-                &mut buf,
-                None,
-            )) as Box<dyn swc_ecma_codegen::text_writer::WriteJs>;
-
-            wr = Box::new(swc_ecma_codegen::text_writer::omit_trailing_semi(wr));
+            let wr = swc_ecma_codegen::text_writer::JsWriter::new(cm.clone(), "\n", &mut buf, None);
+
+            let wr = Box::new(swc_ecma_codegen::text_writer::omit_trailing_semi(wr));
```

Comment:
Boxing it twice wasn't necessary.
Happy to clean this up by importing `JsWriter` and `omit_trailing_semi`? Was trying not to change too much.